.github/workflows/pull_request.yaml

```diff
@@ -63,4 +68,4 @@ jobs:
           make install-deps
       - name: Run tests
         run: |
-          go test ./...
\ No newline at end of file
+          REMOTE_CLUSTER_CONTEXT=kind-skv2-test-remote go test ./...
```

Comment:
Should we maybe run the kind script from within the test somehow so that the context name isn't a magic string from inside of the shell script

Reply:
Going to make the remote cluster context value configurable such that this string is passed to both the setup script and this environment variable. This allows us to be less prescriptive about how users set up their local test environments / manage their kube contexts while relaxing the magic here.